lib/internal/url.js

```diff
@@ -1315,6 +1336,6 @@ module.exports = {
   domainToASCII,
   domainToUnicode,
   urlToHttpOptions,
-  searchParamsSymbol: searchParams,
   encodeStr,
+  isURLThis,
```

Comment:
If we want to use `isURLThis` outside this file, we should give it a better name, like `isURLObject` or `isURL`. `isURLThis` is a big of an odd name.

Reply:
True!